qemu/tests/cfg/multi_host.cfg

```diff
@@ -209,6 +209,77 @@
                       kdump_propagate_cmd = "kdumpctl propagate"
                       kdump_restart_cmd = "systemctl restart kdump.service"
                       kdump_status_cmd = "systemctl status kdump.service"
+                - with_xbzrle:
+                    type = migration_multi_host_with_xbzrle
+                    capabilitys = "xbzrle cache_size"
+                    capabilitys_state = "disable disable"
```

Comment:
I don't understand, we don't have cache_size in capabilities list, how it can be disabled? It's just set by migrate_set_cache_size with integers, and you wont get cache_size value by 'query-migrate-capabilities'. How it can be checked in your capabilitys set?